packages/sources/src/components/dropdown-menu/dropdown-menu.stories.ts

```diff
@@ -30,6 +30,39 @@ export function storiesOfDropdownMenu<TemplateFnReturnType>(
   stories.add("Dropdown Menu", template, {
     args: {
       dropdownAlign: "left",
+      children: `
```

Comment:
Bijna..

Dit komt door de gebrekkige typings van Storybook. Volgens de interface van `DropdownArgs` is `children: TemplateFnReturnType`, maar je wijst hier `string` toe.

Je kan dit afdwingen met de functie `componentArgs()` uit `stories-helpers.ts`. Kijk maar eens hoe we dat bij `tile.stories.ts` doen. Als het goed is komt er dan een error naar voren (cannot assign type string to TemplateFnReturnType oid).

De error los je vervolgens op door deze string door de tagged template literal `html` te laten verwerken, maar dat had je zelf ook al wel door denk ik zo

Reply:
`DropdownArgs` bestaat niet, maar neem aan dat je `DropdownMenuArgs` bedoelt. Dat van dit generieke interface het type argument `TemplateFnReturnType` heet doet niet ter zake, dat kan (en dat is ook beter) net zo goet `T` genoemd worden. 
Anyhow, ik had volgens mij `alert` als voorbeeld genomen en vroeg me aan het eind af waarom ik al type typings toegevoegd had om uiteindelijk op`children:any` uit te komen. `tile` ziet er inderdaad anders uit, dus daar ga ik nu eens naar kijken.

Reply:
Dat het beter `T` kan heten is smaak en aangezien Storybook het ook `TemplateFnReturnType` noemt heb ik het door de rest van het project aangehouden. Lijkt mij niet verstand daarvan af te wijken.

Betekent natuurlijk niet dat we het niet anders kunnen doen, maar ik heb voorkeur om 't overal hetzelfde te doen - ook al blijkt het "verkeerd" te zijn.